dist_tables/range_distribution.rst

```diff
@@ -38,11 +38,11 @@ Next, you can use the master_create_distributed_table() function to mark the tab
 
 This function informs Citus that the github_events table should be distributed by range on the repo_id column.
 
-Range distribution signifies to the database that all the shards have non-overlapping ranges of the distribution key. Currently, the \stage command for data loading does not impose that the shards have non-overlapping distribution key ranges. Hence, the user needs to make sure that the shards don't overlap.
+Range distribution signifies to the database that all the shards have non-overlapping ranges of the distribution key. Currently, the \\copy command for data loading does not impose that the shards have non-overlapping distribution key ranges. Hence, the user needs to make sure that the shards don't overlap.
```

Comment:
That wouldn't be sufficient though? If someone just does a create_distributed_table() with 'range', and tries to COPY, it would presumably error out since no shards exist. So would we have to guide people to do a ``master_create_empty_shard``, manually update the range, and then COPY?

We've been debating removing RANGE entirely, and wondering if we should bite the bullet now.